css-positioning/css-positioning/relative.css

```diff
@@ -0,0 +1,35 @@
+.pokemon {
+  width: 900px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.pokemon > img {
+  width: 150px;
+
+  /* your code here */
+}
+
+#bulbasaur {
+  position: relative;
+}
+
+#charmander {
+  position: relative;
+  top: 20px;
+}
+
+#squirtle {
+  position: relative;
+  top: 60px;
+}
+
+#pikachu {
+  position: relative;
+  top: 140px;
+}
+
+#jigglypuff {
+  position: relative;
```

Comment:
instead of adding `position: relative` to all the pokemon IDs, you can just add it to the `.pokemon > img` selector!